static/js/main.js

```diff
@@ -1,48 +1,42 @@
-document.addEventListener("DOMContentLoaded", function(){
-  var toggle = document.getElementById("scheme-toggle");
-
-  var scheme = "light";
-  var savedScheme = localStorage.getItem("scheme");
-
-  var container = document.getElementsByTagName("html")[0];
-  var prefersDark = window.matchMedia("(prefers-color-scheme: dark)").matches;
-
-  if (prefersDark) {
-    scheme = "dark";
-  }
+function toggleDarkMode() {
```

Comment:
Expected 'use strict' before 'var'. (expected_a_before_b)jslint(expected_a_before_b)

Reply:
What jslint are you using? I did not receive this message.

Reply:
```bash
❯ jslint --version
node-jslint version: 0.12.1  JSLint edition 2013-08-26
```